docs/server/source/production-deployment-template/node-on-kubernetes.rst

```diff
@@ -487,7 +487,119 @@ Step 11: Start a Kubernetes StatefulSet for MongoDB
        $ kubectl --context k8s-bdb-test-cluster-0 get pods -w
   
 
-Step 12: Start a Kubernetes Deployment for MongoDB Monitoring Agent
+Step 12: Configure Users and Access Control for MongoDB
+-------------------------------------------------------
+
+  * Create a user on MongoDB with authorization to create more users and assign
+    roles to them.
+    Note: You need to do this only when setting up the first MongoDB node of
+    the cluster.
+    Log in to the MongoDB instance and open a mongo shell using the certificates
```

Comment:
Maybe add a blank line before this sentence so it doesn't seem like it's part of the "Note:" (unless it is part of the "Note:").